apps/binding.go

```diff
@@ -65,7 +65,7 @@ package apps
 //              "bindings": [
 //                  {
 //                      "location": "send-button",
-//                      "icon": "http://localhost:8080/static/icon.png",
```

Comment:
Should we keep some icons as absolute paths as an example of both?

Reply:
The example should show the idiomatic way of using things. Hence, the icons should all be relative paths as this is the way we encourage devs to use them.